src/test/java/com/bouncestorage/swiftproxy/v1/ObjectResourceTest.java

```diff
@@ -76,6 +76,24 @@ public void testPut() throws Exception {
     }
 
     @Test
+    public void testMissingContainer() throws Exception {
+        Response resp = target.path(Joiner.on("/").join(TestUtils.ACCOUNT_PATH, "foo", "bar"))
+                .request()
+                .header("x-auth-token", authToken)
+                .get();
+        assertThat(resp.getStatus()).isEqualTo(Response.Status.NOT_FOUND.getStatusCode());
+    }
+
+    @Test
+    public void testMissingContainerHEAD() throws Exception {
+        Response resp = target.path(Joiner.on("/").join(TestUtils.ACCOUNT_PATH, "foo", "bar"))
+                .request()
+                .header("x-auth-token", authToken)
+                .head();
+        assertThat(resp.getStatus()).isEqualTo(Response.Status.NOT_FOUND.getStatusCode());
+    }
+
+    @Test
```

Comment:
These kinds of conformance tests are best added in the Swift integration tests -- the unit tests should just do basic sanity checks like in S3Proxy.  Does the former have coverage for this?